src/osemgrep/networking/Scan_helper.ml

```diff
@@ -115,3 +115,106 @@ let report_failure ~dry_run ~token ~scan_id exit_code =
     | Error (code, msg) ->
         Error
           ("API server returned " ^ string_of_int code ^ ", this error: " ^ msg)
+
+(* TODO the server reply would be specified in ATD *)
```

Comment:
do you know to which endpoints this refers too? Might be nice to add also in the comment the endpoint.